kalite/version.py

```diff
@@ -5,416 +9,18 @@
-        "release_date": "2014/01/06",
-        "git_commit": "6e56d05e6f53661aea433d72ce7bddacacddc4b8",
-        "new_features": {
-            "all": [],
-            "students": [
-                "Better performance (faster save times and login times)",
-                "Integration with latest exercises",
-            ],
-            "coaches": [],
-            "admins": [
-                "Easier updates for subtitles",
-            ],
-        },
-        "bugs_fixed": {
-            "all": [],
-            "students": [],
-            "coaches": [],
-            "admins": [],
-        },
-    },
+def load_yaml(file_name):
+    """Creates a dictionary from an yaml file.
 
-    "0.10.2": {
-        "release_date": "2013/10/08",
-        "git_commit": "5831abb8d2ee0815416a17885790679c4672bf97",
-        "new_features": {
-            "all": [],
-            "students": ["Import your KA progress into KA Lite"],
-            "coaches": [],
-            "admins": ["Now start KA Lite by double-clicking the start script"],
-        },
-        "bugs_fixed": {
-            "all": [],
-            "students": [],
-            "coaches": [],
-            "admins": [],
-        },
-    },
+    Args:
+        file_name: The name of the file to be loaded.
 
-    "0.10.1": {
-        "release_date": "2013/09/16",
-        "git_commit": "f048c11289059f36af0bd1c368f9e0fb354b49b5",
-        "new_features": {
-            "all": ["we have a new (online) website"],
-            "students": [],
-            "coaches": [],
-            "admins": [],
-        },
-        "bugs_fixed": {
-            "all": ["better performance"],
-            "students": [],
-            "coaches": [],
-            "admins": [],
-        },
-    },
+    Returns:
+        A dictionary structure that reflects the yaml structure.
+    """
+    with open(file_name, "r") as f:
+        return yaml.load(f)
 
-    "0.10.0": {
-        "release_date": "2013/08/26",
-        "git_commit": "d375134f2f9f1e0cc3cc1bed73227dcecf1061f3",
-        "new_features": {
-            "all": [],
-            "students": ["summary of progress", "video available indicators"],
-            "coaches": ["extended coach reports", "online coach reports"],
-            "admins": ["online usage reports", "synchronization reports"],
-        },
-        "bugs_fixed": {
-            "all": [],
-            "students": [],
-            "coaches": [],
-            "admins": [],
-        },
-    },
 
-    "0.9.2": {
-        "release_date": "2013/02/09",
-        "git_commit": "7c326329b7ae1b6000d1c636cf8ca920a8cc8daa",
-        "new_features": {
-            "all": [],
-            "students": [],
-            "coaches": [],
-            "admins": [],
-        },
-        "bugs_fixed": {
-            "all": [],
-            "students": [],
-            "coaches": [],
-            "admins": [],
-        },
-    },
-}
+VERSION_INFO = load_yaml(os.path.join(os.path.dirname(__file__), "../data/version.yml"))
```

Comment:
couple of notes here:

- don't use os-specific path separators, use the `os.path.join` fully instead :) so change this to something like `os.path.join(dirname(file), "..", "data", "version.yml")`.

- @benjaoming mentioned that looking for the version.yml relative to this file is a bad idea, but right now we import version.py inside settings, so importing settings (where DATA_PATH is defined) would cause a circular import. Not sure what's the best solution, but I'm willing to merge this in for now until we find a better way.